src/pages/projects/ProjectJSON.tsx

```diff
@@ -1,30 +1,39 @@
-export default {
-    "cards": [
-        {
-            "title": "Lucy's Love Bus",
-            "paragraph": `Lucy’s Love Bus improves quality of life for children with cancer
+export interface Project {
+  readonly title: string;
+  readonly paragraph: string;
+  readonly appDesc: string;
+  readonly path: string;
+  readonly abbr: string;
+  readonly learnMore: string;
+}
+
+export const projects : Project[] = [
```

Comment:
⛏️ why is there a space after projects? Also on line 12 there are double quotes.